test/remote/gateways/remote_shift4_v2_test.rb

```diff
@@ -77,4 +77,12 @@ def test_successful_stored_credentials_merchant_initiated
     assert_equal 'merchant_initiated', response.params['type']
     assert_match CHARGE_ID_REGEX, response.authorization
   end
+
+  def test_failed_authorize
```

Comment:
💬 Comment:

Unless you have some custom behavior that I can't see, basically you are testing ruby inheritance here.

Reply:
the idea was to test this with the shift4_v2 credentials.